services/consensusalgo/leanhelixconsensus/test/harness.go

```diff
@@ -83,12 +114,12 @@ func (h *harness) getCommitteeWithNodeIndexAsLeader(nodeIndex int) []primitives.
 	return res
 }
 
-func (h *harness) beLastInCommittee() {
-	h.expectConsensusContextRequestOrderingCommittee(1)
+func (h *harness) dontBeFirstInCommitee() {
+	h.expectConsensusContextRequestOrderingCommittee((h.myNodeIndex() + 1) % NETWORK_SIZE)
```

Comment:
I did this to better reflect what the (previously called) `beLastInCommittee` was doing - which was to make you the second in the committee. But since it was dependent on the node index, I could only commit to it not being the first.

I agree that the naming can be better - will go with the simple `leaderInFirstView` and `validatorInFirstView`